AntWars/Board/Ants/Ant.cs

```diff
@@ -78,7 +95,33 @@ public bool moveUp()
         public bool moveDown()
         {
             Coordinates newCoords = new Coordinates(Coords.X, Coords.Y + 1);
-            return board.BoardObjects.move(this, newCoords);
+            return move(newCoords);
+        }
+
+        private bool move(Coordinates to)
+        {
+            if (!canMove())
+            {
+                die();
```

Comment:
wenn man move öfters aufruft, wird das auch öfters in die dying ants hinzugefügt, das sollte nicht möglich sein. du kannst statt einer liste dort ein ``HashSet`` benutzen dort können kein doppelten vorkommen, sonst solltest du evtl ein flag wie ``isDead`` hinzufügen und den dann überprüfen